plugins/lint-staged-npm/src/index.ts

```diff
@@ -4,14 +4,26 @@ import { getOptions } from '@dotcom-tool-kit/options'
 class TestStaged extends LintStagedHook {
   static description = 'format prettier'
 
-  key = getOptions('@dotcom-tool-kit/lint-staged-npm')?.testGlob ?? '**/*.js'
+  _key?: string
+  get key(): string {
+    if (!this._key) {
+      this._key = getOptions('@dotcom-tool-kit/lint-staged-npm')?.testGlob ?? '**/*.js'
+    }
+    return this._key
+  }
```

Comment:
**suggestion**:

```suggestion
  get key(): string {
    return this._key ||= getOptions('@dotcom-tool-kit/lint-staged-npm')?.testGlob ?? '**/*.js'
  }
```

maybe?

Reply:
I've never seen that operator before!

Reply:
i mean it's less than ideal because it's for booleans but `??=` doesn't exist so

Reply:
actually that's what I used in the end! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment

Reply:
WHATTTT

Reply:
TIL

Reply:
Transpiled in node 12 but still

Reply:
beautiful
```js
return ((_a = this._key) !== null && _a !== void 0 ? _a : (this._key = (_c = (_b = (0, options_1.getOptions)('@dotcom-tool-kit/lint-staged-npm')) === null || _b === void 0 ? void 0 : _b.testGlob) !== null && _c !== void 0 ? _c : '**/*.js'));
```

Reply:
🤮